charts/nri-bundle/Chart.yaml

```diff
@@ -1,16 +1,93 @@
-apiVersion: v1
-appVersion: "1.0"
-description: A Helm chart to deploy New Relic integrations bundled together
+apiVersion: v2
 name: nri-bundle
-version: 3.6.2
+description: A chart groups together the individual charts for the New Relic Kubernetes solution for more comfortable deployment.
 home: https://github.com/newrelic/helm-charts
 icon: https://newrelic.com/assets/newrelic/source/NewRelic-logo-square.svg
+sources:
+  - https://github.com/newrelic/nri-bundle/
+  - https://github.com/newrelic/nri-bundle/tree/master/charts/nri-bundle
+  - https://github.com/newrelic/nri-kubernetes/tree/master/charts/newrelic-infrastructure
+  - https://github.com/newrelic/nri-prometheus/tree/master/charts/nri-prometheus
+  - https://github.com/newrelic/k8s-metadata-injection/tree/master/charts/nri-metadata-injection
+  - https://github.com/newrelic/newrelic-k8s-metrics-adapter/tree/master/charts/newrelic-k8s-metrics-adapter
+  - https://github.com/newrelic/nri-kube-events/tree/master/charts/nri-kube-events
+  - https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-logging
+  - https://github.com/newrelic/helm-charts/tree/master/charts/newrelic-pixie
+  - https://github.com/newrelic/newrelic-infra-operator/tree/master/charts/newrelic-infra-operator
+
+version: 4.3.2
```

Comment:
why 4.3.2 and not 4.0.0 if we are bumping mayor?

Reply:
There are previous updates of the `nri-bundle` chart marked as `devel` that where released previously in the `newrelic-forks` org.

Check it out here: https://github.com/newrelic-forks/helm-charts/commit/3cb9f63bd62a548e8e875bf5ce7214ca90c65598

Reply:
I agree, I would keep with  4.3.2

Reply:
totally miss that, thanks